src/test/kotlin/no/nav/familie/ef/iverksett/tilstand/LagreTilstandJdbcTest.kt

```diff
@@ -0,0 +1,74 @@
+package no.nav.familie.ef.iverksett.tilstand
+
+import no.nav.familie.ef.iverksett.ServerTest
+import no.nav.familie.ef.iverksett.domene.*
+import no.nav.familie.ef.iverksett.tilstand.lagre.LagreTilstandJdbc
+import no.nav.familie.kontrakter.felles.oppdrag.OppdragStatus
+import org.junit.jupiter.api.MethodOrderer
+import org.junit.jupiter.api.Order
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestMethodOrder
+import org.springframework.beans.factory.annotation.Autowired
+import java.time.LocalDate
+import java.util.*
+
+@TestMethodOrder(MethodOrderer.OrderAnnotation::class)
+internal class LagreTilstandJdbcTest : ServerTest() {
+
+    @Autowired
+    private lateinit var lagreTilstandServiceJdbc: LagreTilstandJdbc
+
+    val behandlingsId = UUID.randomUUID()
+    val journalpostId = UUID.randomUUID()
+
+    @Test
+    @Order(1)
+    fun `lagre tilkjentytelse, forvent ingen unntak`() {
+        val tilkjentYtelse = TilkjentYtelse(
+            id = behandlingsId,
+            utbetalingsoppdrag = null,
+            andelerTilkjentYtelse = listOf(
+                AndelTilkjentYtelse(
+                    periodebeløp = Periodebeløp(
+                        beløp = 100,
+                        Periodetype.MÅNED,
+                        fraOgMed = LocalDate.now(),
+                        tilOgMed = LocalDate.now().plusMonths(1)
+                    ),
+                    periodeId = 1L,
+                    forrigePeriodeId = 1L,
+                    kildeBehandlingId = behandlingsId
+                )
+            )
+        )
+        lagreTilstandServiceJdbc.lagreTilkjentYtelseForUtbetaling(behandlingsId, tilkjentYtelse)
```

Comment:
Obs: Her kan vi skape oss litt kaos for fremtiden: 
1) Alltid null ut _alle_ tabellene i `ServerTest` i metoden `fun reset() { .. } `
2) Hver test må kunne kjøres uavhengig av rekkefølge - hver test bør inserte data fra scratch (det som trengs) for å sette opp - som følge av #1 

:)